rizza/helpers/config.py

```diff
@@ -28,14 +28,14 @@ class Config():
     def __attrs_post_init__(self):
         """Load in config files, then environment variables"""
         # first, attempt to load nailgun config
-        self.base_dir = Path.home().joinpath('rizza')
+        self.base_dir = Path(os.getcwd())
```

Comment:
Rizza is meant to operate out of a user's home directory, not current directory

Reply:
I did this just to set up my project. With this upgrade, it worked for me outside the home's directory.